src/main/kotlin/com/depromeet/makers/domain/usecase/UpdateDefaultMemberPassCord.kt

```diff
@@ -0,0 +1,27 @@
+package com.depromeet.makers.domain.usecase
+
+import com.depromeet.makers.domain.exception.PassCordAlreadySetException
+import com.depromeet.makers.domain.gateway.MemberGateway
+import com.depromeet.makers.domain.model.Member
+
+class UpdateDefaultMemberPassCord(
+    private val memberGateway: MemberGateway,
+    private val updateMemberPassCord: UpdateMemberPassCord,
+) : UseCase<UpdateDefaultMemberPassCord.UpdateDefaultMemberPassCordInput, Member> {
+    data class UpdateDefaultMemberPassCordInput(
+        val memberId: String,
+        val passCord: String,
+    )
+
+    override fun execute(input: UpdateDefaultMemberPassCordInput): Member {
+        val member = memberGateway.getById(input.memberId)
+        if (member.hasPassCord()) throw PassCordAlreadySetException()
```

Comment:
```suggestion
        if (isInitializePassCord(member)) throw PassCordAlreadySetException()
```
요부분은 메서드 네이밍 주면 좋을 것 같아여

Reply:
좋습니다!!